### ISSUE TITLE
undefined method `git_editor_comand' for GitReflow:Module

### ISSUE THREAD

**Issue by alexeyraspopov** [reenhanced/gitreflow#176]
Updated to 0.8.0 and found this.

```
~/oss/dgelong test-feature-2
λ git reflow review
git fetch origin master
From github.com:alexeyraspopov/dgelong
 * branch            master     -> FETCH_HEAD

git push origin test-feature-2
Everything up-to-date

[error]
Error: #<NoMethodError: undefined method `git_editor_comand' for GitReflow:Module>
```

**Comment by codenamev** [comment]
Sorry about that.  All fixed in v0.8.1 with above mentioned commit.

**Comment by alexeyraspopov** [comment]
@codenamev cool! thanks for quick fix.
